docs/compatibility.md

```diff
@@ -271,13 +271,66 @@ Spark stores timestamps internally relative to the JVM time zone.  Converting an
 between time zones is not currently supported on the GPU. Therefore operations involving timestamps
 will only be GPU-accelerated if the time zone used by the JVM is UTC.
 
-## Window Functions
+## Windowing
+
+### Window Functions
 
 Because of ordering differences between the CPU and the GPU window functions especially row based
 window functions like `row_number`, `lead`, and `lag` can produce different results if the ordering
 includes both `-0.0` and `0.0`, or if the ordering is ambiguous. Spark can produce different results
 from one run to another if the ordering is ambiguous on a window function too.
 
+### Range Window
+
+When the order-by column of a range based window is numeric type like `byte/short/int/long` and the range boundary calculated for a value has overflow, CPU and GPU will get different results.
+
+For example, let's assume we have two columns, one partition-by column named `uid` and the other order-by column named `dollars`. And the dataset is as below.
```

Comment:
IMO we don't need to elaborate on partition ID and order-by, that's already covered in the SQL statement below.
```suggestion
For example, consider the following dataset:
```

Reply:
Done